frontend/webapp/views/main/main_view.py

```diff
@@ -57,42 +52,18 @@ def display_hijack():
     _mitigation_flag = False
 
     hijack_data = get_hijack_by_key(_key)
-    _configured = False
 
     if hijack_data is None:
         app.artemis_logger.debug("Hijack with id found: {}".format(_key))
         return render_template("404.htm")
 
-    if "configured_prefix" in hijack_data:
-        if (
-            hijack_data["configured_prefix"]
-            in app.config["configuration"].get_prefixes_list()
-        ):
-            _configured = True
-
     if mitigation_status_request.is_up_or_running("mitigation"):
         _mitigation_flag = True
 
     return render_template(
         "hijack.htm",
         data=json.dumps(hijack_data),
         mitigate=_mitigation_flag,
-        configured=_configured,
-        js_version=app.config["JS_VERSION"],
-    )
-
-
-@main.route("/visualizations/", methods=["GET"])
```

Comment:
this is removed until further notice (needs to become more scalable as already described)